packages/components/src/logos/jetpack-logo/index.tsx

```diff
@@ -81,7 +88,6 @@ const JetpackLogo: React.FunctionComponent< JetpackLogoProps > = ( {
 
 	if ( 24 === size ) {
 		return (
-			// eslint-disable-next-line wpcalypso/jsx-classname-namespace
```

Comment:
Nothing happens when I remove this disable comment 🤔

Reply:
Looks like an old artifact from when it actually made a difference.